examples/deepseek_v2/train_deepseekv2.sh

```diff
@@ -188,6 +190,7 @@ NUM_SHARED_EXPERTS=2
 MOE_LAYER_FREQ=1
 
 moe_options=" \
```

Comment:
please refactor this code so as to avoid code duplications. Only the unique stuff should be in the if/else blocks